### PR TITLE
Documented/polished pokemon_summary_screen.c

### PR DESCRIPTION
**Pull request by arantonitis**
Mostly variables mislabeled as global and some formerly unknown stuff.

### CODE REVIEW

**Comment by arantonitis** [comment]
First PR, so please let me know if I've made mistakes with style/convention for the codebase :)

File: src/pokemon_summary_screen.c [hunk 1]
```diff
@@ -97,24 +97,24 @@ static EWRAM_DATA struct PssData
...
-    s16 unk40F0;
+    s16 switchCounter; // Used for various switch statement cases that decompress/load graphics or pokemon data
     u8 unk_filler4[6];
 } *pssData = NULL;
-EWRAM_DATA u8 gUnknown_0203CF20 = 0;
-static EWRAM_DATA u8 gUnknown_0203CF21 = 0;
-static EWRAM_DATA u8 gFiller_0203CF22[2] = {0};
+EWRAM_DATA u8 gLastViewedMonIndex = 0;
+static EWRAM_DATA u8 sUnknown_0203CF21 = 0;
+static EWRAM_DATA u8 sFiller_0203CF22[2] = {0};
```
Comment (nullableVoidPtr) [inline src/pokemon_summary_screen.c]: This looks like `ALIGNED(4)`, i.e. `ALIGNED(4) static EWRAM_DATA u8 sUnknownTaskId = 0;` Try to clean up 'filler' declarations if you can.

File: src/pokemon_summary_screen.c [hunk 2]
```diff
@@ -97,24 +97,24 @@ static EWRAM_DATA struct PssData
...
 static EWRAM_DATA u8 sUnknownTaskId = 0;
-static EWRAM_DATA u8 gFiller_0203CF25[3] = {0};
+static EWRAM_DATA u8 sFiller_0203CF25[3] = {0};
```
Comment (nullableVoidPtr) [inline src/pokemon_summary_screen.c]: Also possibly `ALIGNED(4)`

File: src/pokemon_storage_system.c [hunk 3]
```diff
@@ -1218,7 +1218,7 @@ static const u16 gWallpaperPalettes_Plain[][16] =
 static const u32 gWallpaperTiles_Plain[] = INCBIN_U32("graphics/pokemon_storage/plain.4bpp.lz");
 static const u32 gWallpaperTilemap_Plain[] = INCBIN_U32("graphics/pokemon_storage/plain.bin.lz");
 
-// 12�18 tilemap
+// 12�18 tilemap
```
Comment (nullableVoidPtr) [inline src/pokemon_storage_system.c]: Broken encoding?

Comment (arantonitis) [reply src/pokemon_storage_system.c]: Yeah, I'm not sure what that is?

Comment (nullableVoidPtr) [reply src/pokemon_storage_system.c]: Improperly configured editor perhaps? Just replace the broken Unicode character.

Comment (Phlosioneer) [reply src/pokemon_storage_system.c]: The x was a unicode character, for whatever reason. I had the same problem in another PR. Just replace it with a normal ascii `x`

**Pull request merged**